pkg/i2gw/providers/gce/gateway_converter_test.go

```diff
@@ -156,6 +156,107 @@ var (
 			},
 		},
 	}
+
+	commonHc = gkegatewayv1.CommonHealthCheck{
+		Port: common.PtrTo(testPort),
+	}
+	commonHTTPHc = gkegatewayv1.CommonHTTPHealthCheck{
+		RequestPath: common.PtrTo(testRequestPath),
+	}
+
+	testHealthCheckPolicyName = testServiceName
+	testHealthCheckPolicyHTTP = gkegatewayv1.HealthCheckPolicy{
```

Comment:
It seems like there's a ton of overlap in these health check policies, probably worth adding a helper func to generate them.

Reply:
Yes that's a good suggestion! Updated.